libcylinder/Cargo.toml

```diff
@@ -38,6 +38,7 @@ experimental = [
   # The following features are experimental:
 ]
 
+hash = ["sha2"]
```

Comment:
This feature should be under stable or  experimental, presumably

Reply:
Added to experimental